<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeutraliseSettings</class>
 <widget class="QWidget" name="NeutraliseSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_14" stretch="0,0,0">
     <item>
      <widget class="QLabel" name="ionConcentrationLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>5</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Concentration mol l⁻¹ </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="ionConcentration">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::PlusMinus</enum>
             </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.150000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Positive Ions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="0">
       <widget class="QRadioButton" name="csIon">
        <property name="text">
         <string>Cs⁺</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="liIon">
        <property name="text">
         <string>Li¹⁺</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="rbIon">
        <property name="text">
         <string>Rb⁺</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="mgIon">
        <property name="text">
         <string>Mg²⁺</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="caIon">
        <property name="text">
         <string>Ca²⁺</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="kIon">
        <property name="text">
         <string>K¹⁺</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="naIon">
        <property name="text">
         <string>Na⁺</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Negative Ions</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <widget class="QRadioButton" name="fIon">
        <property name="text">
         <string>F⁻</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="clIon">
        <property name="text">
         <string>Cl⁻</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="brIon">
        <property name="text">
         <string>Br⁻</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="iIon">
        <property name="text">
         <string>I⁻</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
